<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaterWindow</class>
 <widget class="QMainWindow" name="WaterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaterWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadFileBtn">
        <property name="text">
         <string>加载文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lastTimeBtn">
        <property name="text">
         <string>上一时刻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextTimeBtn">
        <property name="text">
         <string>下一时刻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeShower">
        <property name="text">
         <string>当前时刻：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="waterShower" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
